h2o-py/tests/testdir_algos/gbm/pyunit_PUBDEV_4090_grid_model_seeds.py

```diff
@@ -72,7 +72,10 @@ def random_grid_model_seeds_PUBDEV_4090():
     model_len = min(len(air_grid1), len(air_grid2))
     correct_model_seeds = list(range(search_criteria_tune1["seed"], search_criteria_tune1["seed"]+model_len))
     assert model_seeds1[0:model_len]==model_seeds2[0:model_len], "Random gridsearch seed is not set correctly as model seeds..."
-    assert model_seeds1==correct_model_seeds, "Random gridsearch seed is not "
+    # this assert was failing; have to add more diagnostics
+    expected1 = ','.join(correct_model_seeds)
+    actual1 = ','.join(mdel_seeds1)
+    assert model_seeds1==correct_model_seeds, "Model seeds are not correct: expected %s; got %s" % (expected1, actual1) 
 
```

Comment:
Vlad:  I will incorporate your change on my PR.  If you can remove this from yours, that will be great.